lib/PuppeteerSharp/Messaging/RemoteObjectType.cs

```diff
@@ -16,7 +16,7 @@ public enum RemoteObjectType
         /// <summary>
         /// Object.
         /// </summary>
-        Object,
+        ObjectType,
```

Comment:
Hi @kblok  since here there is a change from Object to ObjectType one of the test is failing since it has "JSHandle@object" in puppeteer-sharp\lib\PuppeteerSharp.Tests\PageTests\Events\ConsoleTests.cs---ShouldNotFailForWindowObject()
if you accept this change then we can move forward to change that test string, what do you think this change solved CA1720

Reply:
Let's ignore the rule for this line.

Reply:
Ignored for this line